hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/controller/ContractControllerTest.java

```diff
@@ -179,6 +183,30 @@ void callInvalidValue() {
                 .expectBody(GenericErrorResponse.class);
     }
 
+    @Test
+    void callWithMalformedJsonBody() {
+        webClient.post()
+                .uri(CALL_URI)
+                .contentType(MediaType.APPLICATION_JSON)
+                .body(BodyInserters.fromValue("{from: 0x00000000000000000000000000000000000004e2"))
+                .exchange()
+                .expectStatus()
+                .isEqualTo(BAD_REQUEST)
+                .expectBody(ServerWebInputException.class);
```

Comment:
I'm not sure how this passes, but we should be verifying the payload contents. Same issue for `callInvalidValue()` and `callWithUnsupportedMediaTypeBody()`.
```suggestion
                .expectBody(GenericErrorResponse.class)
                .isEqualTo(new GenericErrorResponse("something");
```